framework/include/mesh/MooseMesh.h

```diff
@@ -1456,6 +1473,12 @@ class MooseMesh : public MooseObject, public Restartable, public PerfGraphInterf
   /// Build lower-d mesh for all sides
   void buildLowerDMesh();
 
+  /// Type of coordinate system per subdomain
+  std::map<SubdomainID, Moose::CoordinateSystemType> _coord_sys;
```

Comment:
do we really support this?

Reply:
we only have a single RZ axis, we only have a single spherical center ((0,0,0) or can it be set?)
so I d say we dont

Reply:
we certainly do. There are tests for it. Trust me I tried